fitbenchmarking/jacobian/base_jacobian.py

```diff
@@ -0,0 +1,30 @@
+"""
+Implements the base class for the Jacobian.
+"""
+from abc import ABCMeta, abstractmethod
+
+
+class Jacobian:
+    r"""
+    Base class for Jacobian.
+    """
```

Comment:
Is the `r"""` meant to be here, or is it a typo?

Reply:
Corrected